projects/media-viewer/src/lib/annotations/comment-set/comment-set-header/comment-search/commet-search.component.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { ToolbarEventService } from '../../../../toolbar/toolbar.module';
+import {CommentSearchComponent} from './comment-search.component';
+
+describe('CommentSearch', () => {
+  let component: CommentSearchComponent;
+  let fixture: ComponentFixture<CommentSearchComponent>;
+  let nativeElement;
+  let toolbarEventService: ToolbarEventService
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [],
+      declarations: [CommentSearchComponent],
+      providers: [ToolbarEventService]
+    }).compileComponents();
+
+    fixture = TestBed.createComponent(CommentSearchComponent);
+    nativeElement = fixture.debugElement.nativeElement;
+    component = fixture.componentInstance;
+    toolbarEventService = new ToolbarEventService();
+    fixture.detectChanges();
+  });
+
+  it('should create the component', () => {
+    expect(component).toBeTruthy();
+  });
+
+
+  it('should emit toggleCommentsSummary event', () => {
+    const commentSummaryButton = nativeElement.querySelector('#commentSummary');
+    commentSummaryButton.click();
+  });
```

Comment:
don't think u are expecting anything here

Reply:
didn't actually touch this, but I'll check when I'm doing the tests